Mail/Views/Thread List/ThreadListCell.swift

```diff
@@ -61,13 +60,19 @@ struct ThreadListCell: View {
         }
         .padding(.leading, multipleSelectionViewModel.isEnabled ? 8 : -4)
         .background(SelectionBackground(isSelected: isSelected, verticalPadding: 2))
+        .background(SelectionBackground(isSelected: selectedThreadBackground, verticalPadding: 0))
         .onTapGesture { didTapCell() }
         .onLongPressGesture(minimumDuration: 0.3) { didLongPressCell() }
         .swipeActions(thread: thread, viewModel: viewModel, multipleSelectionViewModel: multipleSelectionViewModel)
         .clipped()
         .listRowInsets(.init(top: 0, leading: 0, bottom: 0, trailing: 0))
         .listRowSeparator(.hidden)
-        .listRowBackground(cellColor)
+        .listRowBackground(EmptyView())
+        .onChange(of: viewModel.selectedThread) { newThread in
+            if newThread?.uid == thread.uid {
```

Comment:
Same as before, maybe we can directly compare threads ?

Reply:
Same problem as before